### PR TITLE
Adding sap-cop team structure

### LINKED ISSUE #57: Create initial repos and teams for SAP CoP
### Type of repository request

This initial repository will be initialized with the content of the following repo: https://github.com/miminar/sdh-helpers

### Requested name for the repository

sap-datahub

### Repository Description

Utilities and helpers for SAP Data Hub deployment in OpenShift

### Admin Access

sdh-admins

### Write Access

sdh-mergers

### Read Access

NONE

### Additional details

Additionally create the following team structure under redhat-cop Org:

- sap-cop
    - sdh
        - sdh-admins (makentenza, miminar)
        - sdh-mergers (makentenza, miminar)
    - ansible
        - ansible-admins (makentenza, rhmk)
        - ansible-mergers (makentenza, rhmk)

---

Review request:

cc: @redhat-cop/container-cop-core

### PR DESCRIPTION
**Pull request by oybed**
resolves #57 

@makentenza please let us know if this is how you envisioned the team structure to be laid out.

### CODE REVIEW

**Comment by makentenza** [comment]
Thanks @oybed, could you please add 'rhmk' as maintainer in 'sap-cop' and 'ansible' please? And 'miminar' as maintainer as well in 'sdh'? 

Also the team structure is wrong. You have added 'sdh-admins' and 'sdh-mergers' as 'ansible' teams instead 'ansible-admins' and 'ansible-mergers' as requested.

File: config.yaml [hunk 1]
```diff
@@ -1208,6 +1208,59 @@ orgs:
...
+                repos:
+                  sap-datahub: write
+          ansible:
+            description: ""
+            maintainers:
+              - makentenza
+            members:
+            privacy: closed
+            teams:
+              sdh-admins:
```
Comment (makentenza) [inline config.yaml]: This should be 'ansible-admins'

File: config.yaml [hunk 2]
```diff
@@ -1208,6 +1208,59 @@ orgs:
...
+                description: ""
+                maintainers:
+                  - makentenza
+                  - rhmk
+                members:
+                privacy: closed
+              sdh-mergers:
```
Comment (makentenza) [inline config.yaml]: This should be 'ansible-mergers'

**Comment by oybed** [comment]
@makentenza comments incorporated  - please take a look and let us know if this looks right

**Comment by makentenza** [comment]
Thanks! LGTM now @oybed

**Comment by oybed** [comment]
great - @sabre1041 I believe it's ready for your final approval and merge.

**Pull request merged**